junction_monitor:
  ros__parameters:
    no_of_junctions: 2
    junction_1: [0.0, -1.0, 1.0, -1.0, 1.0, 1.0, 0.0, 1.0]
    junction_2: [10.8, -11.0, 10.0, -10.0, 11.0, 10.0, 10.8, 10.0]
